packages/flutter_tools/bin/web_swap.dart

```diff
@@ -0,0 +1,116 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```

Comment:
Would it make sense to put this under `dev/`?

Reply:
Re-using the tools code is too useful I think. Easy to add and remove here